src/main/java/org/terasology/dynamicCities/construction/Construction.java

```diff
@@ -38,8 +38,7 @@
 import org.terasology.dynamicCities.buildings.components.ProductionChestComponent;
 import org.terasology.dynamicCities.buildings.components.SettlementRefComponent;
 import org.terasology.dynamicCities.buildings.events.OnSpawnDynamicStructureEvent;
-import org.terasology.dynamicCities.construction.events.BuildingEntitySpawnedEvent;
-import org.terasology.dynamicCities.construction.events.SpawnStructureBufferedEvent;
+import org.terasology.dynamicCities.construction.events.*;
```

Comment:
Please avoid star imports (in IntelliJ you have to set the number of classes for which start import is triggered to some very high number)